From 74fa68dd6690d0aaed14f2340c9a36bad91c019b Mon Sep 17 00:00:00 2001
From: Firoz Khan <firozk@codeaurora.org>
Date: Tue, 5 Jun 2018 09:41:41 +0530
Subject: [PATCH] arch: microblaze: System call table generator

- Generate include/generated/uapi/asm/unistd.h and
  include/generated/asm/syscalls.h
- Add kernel/syscall.S file which will in turn include
  include/generated/asm/syscalls.h file
---
 arch/microblaze/kernel/syscall.S | 6 ++++++
 1 file changed, 6 insertions(+)
 create mode 100644 arch/microblaze/kernel/syscall.S

diff --git a/arch/microblaze/kernel/syscall.S b/arch/microblaze/kernel/syscall.S
new file mode 100644
index 0000000..71eff9c
--- /dev/null
+++ b/arch/microblaze/kernel/syscall.S
@@ -0,0 +1,6 @@
+/* SPDX-License-Identifier: GPL-2.0 */
+
+#define __SYSCALL(nr, entry, nargs)    .long entry
+	
+ENTRY(sys_call_table)
+#include <asm/syscalls.h>
-- 
1.9.1

